torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1843,6 +1843,24 @@ RegisterOperators reg2({
     DEFINE_INT_OP(aten::__or__, a | b),
     DEFINE_INT_OP(aten::__xor__, a ^ b),
 
+    //all is a function that takes an iterable and returns True if all are True
+      Operator(
+        "aten::all(Tensor x) -> bool",
+        [](Stack& stack) {
+          at::Tensor x;
+          pop(stack, x);
+          for(int i = 0; i < sizeof(x)/sizeof(x[0]); i++){
```

Comment:
There is already an op for `aten::all` in the ops that are generated from the code in `aten`, so I don't think we need this op, just adding it to the builtins list should be enough.